src/backend/internal/migrations/20240130153800_people.up.sql

```diff
@@ -4,12 +4,12 @@ CREATE TABLE users(
     email varchar(2000) UNIQUE NOT NULL,
     provider varchar(255),
     password varchar(2000),
-    logged_out boolean NOT NULL
+    logged_out boolean NOT NULL,
+    profile_picture varchar(255)
```

Comment:
please do change the naming to `profile_picture_key` and extend the length to `varchar(2000)`